controller/lib/core/blower_pid.cpp

```diff
@@ -98,19 +89,19 @@ void blower_pid_execute(const BlowerSystemState &desired_state,
 
   Pressure cur_pressure = get_patient_pressure();
 
-  Setpoint = desired_state.setpoint_pressure.kPa();
-  Input = cur_pressure.kPa();
-  myPID.Compute();
+  float output;
+  myPID.Compute(/*input=*/cur_pressure.kPa(),
+                /*setpoint=*/desired_state.setpoint_pressure.kPa(), &output);
 
   // If the blower is not enabled, immediately shut down the fan.  But for
   // consistency, we still run the PID iteration above.
   if (!desired_state.blower_enabled) {
-    Output = 0;
+    output = 0;
```

Comment:
This is not what your PR is about, but overwriting the PID output while it still computes can lead to integral saturation, as the pressure is no longer following what the PID tries to do, which means that when it is turned back on, the output will probably be wrong.

This means there should be a routine that resets PID.output_sum_ when this condition is met.

Reply:
Do you mean that the PID API should have a method for telling it "I'm gonna do something different than the output you gave me"? Does my PR exacerbate the issue, or should I just put a TODO?

Reply:
The issue was already there in the way we used the existing API, but indeed, the Manual mode gave us a way (that we should have used) to do that.
Calling a reset method that allows us to reset output_sum_ (as well as next_sample_time_ and last_update_time) is the best way to achieve this.